css/styles.css

```diff
@@ -0,0 +1,53 @@
+header {
+  display: flex;
+  background-color: #562b81;
+  color: white;
+  font-family: sans-serif;
+  font-size: 20px;
+  align-items: center;
+}
+
+form {
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+h1 {
+  font-family: sans-serif;
+}
+
+.url-pic {
+  height: 200px;
+  width: 300px;
```

Comment:
We don't want to set both a `height` & a `width` or else we'll distort the image. You should just be doing `width: 100%` so it takes up the full width of it's parent, and the height will automatically adjust accordingly